src/address/entity.rs

```diff
@@ -0,0 +1,33 @@
+use crate::primitives::{bitcoin::*, *};
+use derive_builder::Builder;
+
+pub struct Address {
+    pub id: AddressId,
+    pub address_string: String,
+    pub profile_id: ProfileId,
+    pub keychain_id: KeychainId,
+    pub kind: pg::PgKeychainKind,
+    pub address_idx: u32,
+    pub external_id: Option<String>,
+    pub metadata: Option<serde_json::Value>,
+}
+
+#[derive(Builder, Clone, Debug)]
+pub struct NewAddress {
+    pub id: AddressId,
+    #[builder(setter(into))]
+    pub address_string: String,
```

Comment:
Redundant. Don't name things with the type `address_string: String` is not needed - you are already communicating its a `String`